### ISSUE TITLE
Release recent updates

### ISSUE THREAD

**Issue by olexandr-konovalov** [gap-packages/qaos#7]
Package update scripts produce this:
```
Package: qaos
  changed components: [ "Dependencies", "PackageDoc" ]
  ERROR (qaos): There are changed components in the new info file from
  https://gap-packages.github.io/qaos/PackageInfo.g
   but .Version remains the same. This is not allowed, so the info file will not be changed
  PackageInfo.g is not accepted because of an error.
```
Apparently the new package archive with the same version number has been made to include https://github.com/gap-packages/qaos/commit/59a56a4d64ac6c2c010bec812ef1f7c2dc90b80d and https://github.com/gap-packages/qaos/commit/f3b058bfcde4c38e3034dc0da161ac7a5d70b559. It would be good to produce a new version (also to include https://github.com/gap-packages/qaos/commit/7f57bba6d682e32c985092413c433331b6ded763) and make a new release properly incrementing version number.

**Comment by fingolfin** [comment]
I have now released 1.5, which hopefully fixes this.

**Comment by olexandr-konovalov** [comment]
Thanks - it has been picked up:
```
* qaos - new version 1.5 discovered!!!
  ============================================
  Getting new archives from 
  https://github.com/gap-packages/qaos/releases/download/v1.5/qaos-1.5
[ ".tar.gz" ]
  unpacking qaos-1.5.tar.gz
  Validation of the info file successful!
  Package qaos 1.5 from 30/07/2017 has dependencies:
  * GAP >= 4.8
  * ExternalConditions [ "needs cURL (http://curl.haxx.se)" ]
  finding text files  . . .

=====================text files==========================
qaos-1.5/init.g
qaos-1.5/LICENSE
qaos-1.5/makedoc.g
qaos-1.5/PackageInfo.g
qaos-1.5/read.g
qaos-1.5/README
qaos-1.5/doc/chap0.html
qaos-1.5/doc/chap0.txt
qaos-1.5/doc/chap0_mj.html
qaos-1.5/doc/chap1.html
qaos-1.5/doc/chap1.txt
qaos-1.5/doc/chap1_mj.html
qaos-1.5/doc/chap2.html
qaos-1.5/doc/chap2.txt
qaos-1.5/doc/chap2_mj.html
qaos-1.5/doc/chap3.html
qaos-1.5/doc/chap3.txt
qaos-1.5/doc/chap3_mj.html
qaos-1.5/doc/chapInd.html
qaos-1.5/doc/chapInd.txt
qaos-1.5/doc/chapInd_mj.html
qaos-1.5/doc/chooser.html
qaos-1.5/doc/concept.xml
qaos-1.5/doc/install.xml
qaos-1.5/doc/lefttoc.css
qaos-1.5/doc/manual.css
qaos-1.5/doc/nocolorprompt.css
qaos-1.5/doc/qaos.xml
qaos-1.5/doc/ragged.css
qaos-1.5/doc/times.css
qaos-1.5/doc/title.xml
qaos-1.5/doc/toggless.css
qaos-1.5/doc/user.xml
qaos-1.5/gap/helper.gd
qaos-1.5/gap/helper.gi
qaos-1.5/gap/qaos.gd
qaos-1.5/gap/qaos.gi
qaos-1.5/tst/testall.g

=====================end of the list of text files=======
=====================binary files========================
qaos-1.5/doc/manual.js
qaos-1.5/doc/manual.pdf
qaos-1.5/doc/manual.six
qaos-1.5/doc/rainbow.js
qaos-1.5/doc/toggless.js

=====================end of the list of binary files=====
=====================ignored files=======================

=====================end of the list of ignored files====
```
